rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/AbstractIModeCommand.java

```diff
@@ -46,6 +53,17 @@ public AbstractIModeCommand() {
 
     @Override
     public void execute(ApplicationContext currentContext) throws CLIException {
+        CommandFactory commandFactory = CommandFactory.getCommandFactory(CommandFactory.Type.ALL);
+        AbstractDevice console = null;
+        try {
+            console = AbstractDevice.getConsole(AbstractDevice.JLINE);
+            ((JLineDevice) console).setCommands(ObjectArrays.concat(commandFactory.supportedCommandEntries(),
+                                                                    CommandSet.INTERACTIVE_COMMANDS,
+                                                                    CommandSet.Entry.class));
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1148)